Library/Homebrew/test/support/quiet_progress_formatter.rb

```diff
@@ -0,0 +1,14 @@
+require "rspec/core/formatters/progress_formatter"
+
+class QuietProgressFormatter < RSpec::Core::Formatters::ProgressFormatter
+  RSpec::Core::Formatters.register self, :seed
```

Comment:
Should this include `:message` now?

Reply:
I don't think so. It works without it. From my reading of the docs I think the fact that we're explicitly specifying this `--format` on the command line makes it (and possibly this whole `register()` call) unnecessary.